src/functions-and-arrays.js

```diff
@@ -52,14 +130,43 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  const newArr = []
+  if (wordsUnique.length === 0) {
+    return null
+  }
+  else {
+    for (i = 0; i < wordsUnique.length; i++) {
+      if (newArr.includes(wordsUnique[i]) === false) {
+        newArr.push(wordsUnique[i])
+      }
+    }
+    return newArr
+  }
```

Comment:
Bien resuelto. Nada más comenzar no se espera que encontréis una solución compleja a esta kata pero la forma más elegante (puedes investigarlo) de resolver este problema sería a través de un **Set**: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Set

Un Set  es una estructura de datos que alberga elementos ÚNICOS no repetidos.